applications/accounts/scripts/create_api_user.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+NAMESPACE=${CH_ACCOUNTS_REALM}
+USERNAME=admin_api
+PASSWORD=$(echo $RANDOM | md5sum | head -c 20; echo;)
+
+api_user_secret_exists=$(kubectl -n ${NAMESPACE} get secret accounts-api >/dev/null; echo $?)
+
+if [ "${api_user_secret_exists}" -ne "0" ];
+then
+set -e
+echo Creating API user
+
+# create the user and reload keycloak
+/opt/jboss/keycloak/bin/add-user-keycloak.sh -u ${USERNAME} -p ${PASSWORD}
+
+kubectl create secret generic -n ${NAMESPACE} accounts-api \
```

Comment:
not sure we want to allow this in the pod

Reply:
changed it, see commit below